docs/source/api/pystiche_papers.johnson_alahi_li_2016.rst

```diff
@@ -1,13 +1,32 @@
 ``pystiche_papers.johnson_alahi_li_2016``
 =========================================
 
+Unfortunately, the parameters in the reference implementation differ from the parameters
+described in the paper. If ``impl_params is True``, the parameters from the reference
+implementation are used instead of the parameters from the paper. The following
+functions are affected by the parameter ``impl_params``:
+
+  - the :func:`~pystiche_papers.johnson_alahi_li_2016.training`
+  - the :func:`~pystiche_papers.johnson_alahi_li_2016.stylization`
+  - the :func:`~pystiche_papers.johnson_alahi_li_2016.multi_layer_encoder`
+  - the :func:`~pystiche_papers.johnson_alahi_li_2016.content_transform`
+  - the :func:`~pystiche_papers.johnson_alahi_li_2016.style_transform`
+  - the :func:`~pystiche_papers.johnson_alahi_li_2016.batch_sampler`
+  - the :func:`~pystiche_papers.johnson_alahi_li_2016.transformer`
+  - the :func:`~pystiche_papers.johnson_alahi_li_2016.content_loss`
+  - the :func:`~pystiche_papers.johnson_alahi_li_2016.style_loss`
+  - the :func:`~pystiche_papers.johnson_alahi_li_2016.regularization`
```

Comment:
Is any of the top-level functions not affected? If not, I think this list is not needed.

Reply:
Yes, some of the top-level functions are not affected:

pystiche_papers.johnson_alahi_li_2016.preprocessor
pystiche_papers.johnson_alahi_li_2016.postprocessor
pystiche_papers.johnson_alahi_li_2016.image_loader
pystiche_papers.johnson_alahi_li_2016.images
pystiche_papers.johnson_alahi_li_2016.dataset